src/GraphControl/Control/Grapher.cpp

```diff
@@ -341,7 +336,7 @@ namespace GraphControl
             m_tokenEquationStyleChanged = newer->EquationStyleChanged += ref new EquationChangedEventHandler(this, &Grapher::OnEquationStyleChanged);
         }
 
-        UpdateGraph();
+        OnEquationChanged();
```

Comment:
Why change this? I would rather call the method directly instead of the event handler directly

Reply:
reverted.